lib/services/music_player_background_task.dart

```diff
@@ -281,8 +283,12 @@ class MusicPlayerBackgroundTask extends BaseAudioHandler {
   Future<void> dispose() => _player.dispose();
 
   @override
-  Future<void> play() {
-    return _player.play();
+  Future<void> play() async {
+    if (FinampSettingsHelper.finampSettings.audioFadeOutDuration > Duration()) {
```

Comment:
You should use `Duration.zero` instead of an empty instance (same in the pause handler)

Reply:
Also, you're comparing with `audioFadeOutDuration`, but it should be `audioFadeInDuration`